tinygrad/engine/realize.py

```diff
@@ -61,7 +61,8 @@ def get_kernel(renderer:Renderer, ast:UOp) -> Kernel:
                   raise RuntimeError(f"mismatch of {diff_count}/{b.numel()} items with type {b.dtype}, max {(b-bufs[0]).abs().max().item()}")
   if logkerns is not None: logkerns.writelines([f"{(k.ast, k.applied_opts)}\n"])
   if DEBUG >= 5: print((k.ast, k.applied_opts)) # print here to show final applied_opts for all kernels instead of just in beam_search
-  return k
+  # NOTE: this can return multiple kernels for one ast in the future (for things like split reduce op)
+  return [k]
```

Comment:
I wouldn't put this in this PR. It's not a used feature, and it might actually be one kernel with split